### ISSUE TITLE
photoperiod paper: add to Implications the need to incorporate photoperiod into forecasting

### ISSUE THREAD

**Issue by AileneKane** [lizzieinvancouver/ospree#141]
This includes:
1) Nacho: write a paragraph on how niche/species distribution models deal with (or do not deal with) photoperiod in forecasting
2) Cat: Ask Miriam how ED (and/or other models her lab uses) deal with photoperiod in forecasts
3) Ailene: Add to outline

**Comment by cchambe12** [comment]
I reached out to Miriam and she gave a very thorough reply. I will post the papers she suggested on the Photoperiod Wiki page. Here is her emailed response... I can move it to a readME or to the wiki if we decide that is more useful.

From Miriam:
"I did a bit of looking & asking about this when I ran into Dan last week (the week before?) in HUCE... the paper I know about related to photoperiod in ED is this one, but I'm not confident that my knowledge of this is exhaustive.   

Some searching in the Fortran code suggests that we trace daylength according to latitude & day-of-year & we set 2 leaf photoperiod parameters (min & max daylength) according to Jolly et al. 2005 (GCB). For particular plant functional types, the relationship between daylight & the daylength parameters can affect the "leaf elongation factor" (I say "can" because other things, like dryness, could trump the light effect). Specifically, in my version of ED2, the formula is:

elongf_photop = min(1.0, max(0.0, (daylight - dl_min(ipft))     /          ( dl_max(ipft) - dl_min(ipft) ) ) )

[where "daylight" is the true daylight, "dl_min" and "dl_max" are parameters, and "ipft" is just an index suggesting that these parameters could vary based on plant functional type].

The leaf elongation factor (again, which may or may not be the photoperiod-related version depending on what else is going on) is then used in growth equations, leaf dropping dynamics, etc.

How much detail are you looking for? I'm not so familiar with this part of the model... it's all on GitHub here, in case you're interested in the constellation of stuff affected by the elongation factor. If you do choose to go digging, I suggest starting with ED/src/dynamics/phenology_driv.f90 and then working your way out using "grep" commands [i.e. from the folder of interest, searching recursively: grep -ri "elongf_photop" .]

Best,
Miriam"
